z_infra_provisioning/cloud_infra_center/ocp_upi/roles/configure-compute-nodes/tasks/main.yaml

```diff
@@ -83,4 +82,34 @@
     terminate_volume: True
   with_indexed_items: "{{ [os_compute_server_name] * os_compute_nodes_number }}"
   when:
-  - disk_type == "scsi"
\ No newline at end of file
+  - disk_type == "scsi"
+  - use_storage_provider == false
+
+- name: 'Create BFV compute volume from backend storage provider'
+  os_volume:
+    state: present
+    name: "{{ item.1 }}-{{ item.0 }}-bfv"
+    image: "rhcos"
+    size: "{{ compute_flavor_size.stdout_lines[0]}}"
+    volume_type: "{{ volume_type_template }}"
+    metadata: "{{ cluster_id_tag }}"
+    timeout: 3600
```

Comment:
we have timeout 3600 here
what's the relationship between this 3600 and previous timeout 10 min?

Reply:
thanks for your comments, it's already merged.